src/Builder/Request/OrderBuilder.php

```diff
@@ -145,6 +151,11 @@ public function addCouponData(OrderInterface $order): void
         }
     }
 
+    public function addAdditionalData(OrderInterface $order): void
```

Comment:
Since this PR is going to force us to release a major version, could you submit this feature as a separate PR please? In that case we can also have it for a v1.